Doc/library/re.rst

```diff
@@ -28,7 +28,10 @@ character for the same purpose in string literals; for example, to match
 a literal backslash, one might have to write ``'\\\\'`` as the pattern
 string, because the regular expression must be ``\\``, and each
 backslash must be expressed as ``\\`` inside a regular Python string
-literal.
+literal. Also, please note that any invalid escape sequences in Python's
+usage of the backslash in string literals now generate a DeprecationWarning
```

Comment:
Could you use a link to the DeprecationWarning exception like that 
```
:exc:`DeprecationWarning`
```

same for the SyntaxError exception